AppCenterCrashes/AppCenterCrashesTests/MSCrashesCategoryTests.m

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAppCenterInternal.h"
+#import "MSCrashesCategory.h"
+#import "MSCrashesPrivate.h"
+#import "MSMockUserDefaults.h"
+#import "MSTestFrameworks.h"
+#import "MSUtility+Application.h"
+
+#if TARGET_OS_OSX
+
+@interface NSApplication (CrashException)
+
+- (void)ms_reportException:(NSException *)exception;
+- (void)ms_sendEvent:(NSEvent *)theEvent;
+
+@end
+
+#endif
+
+static NSException *lastException;
+static void exceptionHandler(NSException *exception) { lastException = exception; }
+
+@interface MSCrashesCategoryTests : XCTestCase
+
+@end
+
+@implementation MSCrashesCategoryTests
+
+- (void)testActivateCategory {
```

Comment:
There are many combination of `NSBundle`, `NSUserDefaults` and `isDebuggerAttached` values. I can see test coverage at least for `NSBundle`. Please add more unit tests and also consider splitting into two separate tests for this one.

Reply:
> also consider splitting into two separate tests for this one.

Actually I can't really separate it into separate tests because we can swizzle only once